go.mod

```diff
@@ -31,3 +31,5 @@ require (
 	github.com/zclconf/go-cty-debug v0.0.0-20191215020915-b22d67c1ba0b
 	golang.org/x/tools v0.0.0-20200323144430-8dcfad9e016e
 )
+
+replace github.com/hashicorp/terraform-schema v0.0.0-20210522075401-0f5e258f5e97 => github.com/Nepomuceno/terraform-schema v0.0.0-20210527083206-b43d158a9361
```

Comment:
Once @Nepomuceno changes are merged in the terraform-schema, I'll update go.mod.